tests/jvm/src/test/scala/scala/meta/tests/semanticdb/DatabaseSuite.scala

```diff
@@ -10,13 +10,12 @@ import scala.tools.nsc.reporters.StoreReporter
 import scala.compat.Platform.EOL
 import scala.{meta => m}
 import scala.meta.io._
-import scala.meta.internal.semanticdb.DatabaseOps
-import scala.meta.internal.semanticdb.FailureMode
-import scala.meta.internal.semanticdb.MemberMode
-import scala.meta.internal.semanticdb.SemanticdbMode
+import scala.meta.internal.semanticdb.{DatabaseOps, FailureMode, MemberMode, OverrideMode, SemanticdbMode}
 import scala.meta.testkit.DiffAssertions
 
-abstract class DatabaseSuite(mode: SemanticdbMode, members: MemberMode = MemberMode.None)
+abstract class DatabaseSuite(mode: SemanticdbMode,
+                             members: MemberMode = MemberMode.None,
+                             overrides: OverrideMode = OverrideMode.None)
```

Comment:
Why is override generation disabled by default here? Do we assume that the feature is too risky to even enable it in tests?

Reply:
@xeno-by, MemberMode was turned off, I assumed OverrideMode should be turned off as well. Should I keep it on?

Reply:
@olafurpg Do you remember the motivation behind making members optional in DatabaseSuite?